test/builder/pipelineRun.go

```diff
@@ -143,6 +143,21 @@ func Logging(runID *api.CustomJSON) PipelineRunSpecOp {
 				RunID: runID,
 			},
 		}
+
+		return spec
+	}
+}
+
+// LoggingWithIndexURL creates a PipelineRunSpecOp which adds Logging to the PipelineRun with specific indexURL
```

Comment:
You should only set one aspect of the logging here, not both.
Check is logging is nil -> Create it, else modify only one aspect. See Line 105 as example.
Add tests

Reply:
Rename Logging above to LoggingWithRunID